cmd/operator/main.go

```diff
@@ -282,8 +275,34 @@ func Main() int {
 		return 1
 	}
 
+	var scrapeConfigSupported bool
+	err = cc.CheckPrerequisites(ctx,
+		allowedNamespaces,
+		verbs,
```

Comment:
`verbs` refers to the PrometheusAgent resource?

Reply:
PrometheusAgent resource mapped to corresponding operations.

https://github.com/prometheus-operator/prometheus-operator/blob/4b5aa9f6736d3ef8da7c17fcfbc3bad22d22fd1a/cmd/operator/main.go#L273-L276